recordedit/recordEdit.app.js

```diff
@@ -183,6 +183,15 @@
                 } else {
                     if ($rootScope.reference.canCreate) {
                         $rootScope.displayname = $rootScope.reference.displayname;
+
+                        // populate defaults
+                        angular.forEach($rootScope.reference.columns, function(column) {
+                            // typeof column.default !== "function"
+                            if (column.default) {
```

Comment:
Curious about why you're not checking via `typeof` here, since defaults are only filled if they're not a function and is a value? (And if you don't need this commented line, it should be removed.)